lib/spack/docs/binary_caches.rst

```diff
@@ -1,108 +1,132 @@
 for all of its link and run dependency packages as well. Compressed tarballs are
-signed with gpg and signature and tarball and put in a ".spack" file. Optionally
-, the rpaths ( and ids and deps on macOS ) can be changed to paths relative to 
-the spack install tree before the tarball is created.
+signed with gpg and signature and tarball and put in a ``.spack`` file. Optionally,
+the rpaths (and ids and deps on macOS) can be changed to paths relative to
+the Spack install tree before the tarball is created.
 
 Build caches are created via:
 
-.. code-block:: sh
+.. code-block:: console
 
-   $ spack buildcache create 
+   $ spack buildcache create
 
 
+---------------------------------------
 Finding or installing build cache files
 ---------------------------------------
 
-To find build caches or install build caches, a spack mirror must be configured
-with
- 
-``spack mirror add <name> <url>``. 
+To find build caches or install build caches, a Spack mirror must be configured
+with:
+
+.. code-block:: console
+
+   $ spack mirror add <name> <url>
 
-Build caches are found via: 
+Build caches are found via:
 
-.. code-block:: sh
+.. code-block:: console
 
    $ spack buildcache list
 
 Build caches are installed via:
 
-.. code-block:: sh
+.. code-block:: console
 
-   $ spack buildcache install 
-   
+   $ spack buildcache install
 
+
+----------
 Relocation
 ----------
 
-Initial build and later installation do not necessarily happen at the same 
-location. Spack provides a relocation capability and corrects for RPATHs and 
-non-relocatable scripts. However, many packages compile paths into binary 
-artificats directly. In such cases, the build instructions of this package would
+Initial build and later installation do not necessarily happen at the same
+location. Spack provides a relocation capability and corrects for RPATHs and
+non-relocatable scripts. However, many packages compile paths into binary
+artifacts directly. In such cases, the build instructions of this package would
 need to be adjusted for better re-locatability.
 
+.. _cmd-spack-buildcache:
 
-Usage 
------
-spack buildcache create <>
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-Create tarball of installed spack package and all dependencies. 
-Tarballs is checksummed and signed if gpg2 is available.
-Places them in a directory build_cache that can be copied to a mirror.
-Commands like "spack buildcache install" will search it for pre-compiled packages.
- 
-
-options:
-
--d <path> : directory in which "build_cache" direcory is created, defaults to "."
--f : overwrite ".spack" file in "build_cache" directory if it exists
--k <key> : the key to sign package with. In the case where multiple keys exist, the package will be unsigned unless -k is used.
--r : make paths in binaries relative before creating tarball
--y : answer yes to all create unsigned "build_cache" questions
-<> : list of package specs or package hashes with leading /
```

Comment:
Previously, this entire section was on a single line: http://spack.readthedocs.io/en/latest/binary_caches.html#spack-buildcache-create

Reformatted to use a table.